foundations/installations/installations.md

```diff
@@ -28,185 +28,180 @@ Installing a VM is a simple process. This guide uses Oracle's VirtualBox program
 
 #### IMPORTANT
 
-Once you have completed these instructions, **you are expected to work entirely in the VM.** Maximize the window, add more virtual monitors if you have them, fire up the Internet Browser in the **Whisker Menu** ![The Whisker Menu Icon](https://cdn.statically.io/gh/TheOdinProject/curriculum/5d27ddb08c8cf3c553537deb6156a5c7f7aa1bac/foundations/installations/prerequisites/imgs/whisker_menu_icon.png) on the top left of the desktop. You should not be using anything outside of the VM while working on The Odin Project. If you feel like you have a good understanding after using the VM for a while, and or want to improve your experience, we recommend dual-booting Ubuntu, which there are instructions for below.
+Once you have completed these instructions, **you are expected to work entirely in the VM.** Maximize the window, add more virtual monitors if you have them, fire up the Internet Browser in the **Whisker Menu** <img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/5d27ddb08c8cf3c553537deb6156a5c7f7aa1bac/foundations/installations/prerequisites/imgs/whisker_menu_icon.png" alt="The blue-white rodent Whisker Menu Icon" style="display: inline; margin: auto;"> on the top left of the desktop. You should not be using anything outside of the VM while working on The Odin Project. If you feel like you have a good understanding after using the VM for a while, and or want to improve your experience, we recommend dual-booting Ubuntu, which there are instructions for below.
```

Comment:
Yep, otherwise it just breaks the flow, examples of how it currently looks on site:
![image](https://user-images.githubusercontent.com/95084627/211091092-5a603d9c-2ad6-4be0-9435-f347b1c9ab63.png)
![image](https://user-images.githubusercontent.com/95084627/211091101-867fe7ea-2c66-423c-98f2-53336b5857d4.png)

In this case, it ain't terrible but it is not good either. Inline looks a lot better & the FAQ section with the VBox icons was awful with block styling.

Reply:
Sounds good :)